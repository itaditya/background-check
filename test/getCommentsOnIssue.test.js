const getCommentsOnIssue = require('../lib/github-api/getCommentsOnIssue')

test('getCommentsOnIssue is working', async () => {
  const github = {
    issues: { getComments: jest.fn() }
  }
  await getCommentsOnIssue(github, {
    owner: 'itaditya',
    repo: 'private-gh-app-test-repo',
    issueNum: 1
  })

  expect(github.issues.getComments.mock.calls).toMatchSnapshot()
})
