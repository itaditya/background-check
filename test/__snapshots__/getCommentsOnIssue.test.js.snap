// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCommentsOnIssue is working 1`] = `
Array [
  Array [
    Object {
      "number": 1,
      "owner": "itaditya",
      "repo": "private-gh-app-test-repo",
    },
  ],
]
`;
